src/main/resources/config.yml

```
@@ -55,6 +55,14 @@ settings:
         - tag:shulker_boxes
     # When enabled, name tags will be hidden from all players in the game
     hide-nametags: true
+    #enables an item to click to leave game while waiting
+    enable-leave-item: true
+    #enables a item to force start game for people with hg.forcestart perm
+    enable-force-start-item: true
+    #item that is shown for leave
+    leave-item-type: RED_BED
+    #item that is shown to force start game
+    force-start-item-type: NETHER_STAR
```

spacing and capitalization for consistency
```suggestion
    # Enables an item to click to leave game while waiting
    enable-leave-item: true
    # Enables a item to force start game for people with "hg.forcestart" perm
    enable-force-start-item: true
    # Item that is shown for leave
    leave-item-type: RED_BED
    # Item that is shown to force start game
    force-start-item-type: NETHER_STAR
```